src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  bookList: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    add: (state, action) => {
+      const {title, author} = action.payload;
+      const newBook = {
+        category: 'Undefined',
+        title: title,
+        author: author,
+      }
```

Comment:
- [OPTIONAL] Consider assigning a unique identifier (`id`) property to each new book. This will facilitate future operations like removing or editing specific books.

```suggestion
      const newBook = {
        category: 'Undefined',
        title: title,
        author: author,
        id: Math.floor(Math.random() * 1000),
      }
```